src/lib/SlideBarChart.tsx

```diff
@@ -263,19 +267,19 @@ class SlideBarChart extends Component<SlideBarChartComponentProps, State> {
    * so we throttle the events here to prevent too many calls being sent across
    * the bridge which we have seen to cause lag on slow Android devices
    */
-  moveCursorBinary(value: number, isTouch = false) {
+  moveCursorBinary(value: number, isTouch = false, isFirstMove = false) {
```

Comment:
Instead of passing in the `isFirstMove` parameter, it would be nice to simply pass the correct `value` parameter. 
So similar to `getInitialIndicatorPosition` in the area chart component, we could have a function to calculate the input to `value` from `initialBarSelected` and the dimensions of the chart.